client/ayon_blender/plugins/create/create_camera.py

```diff
@@ -40,3 +41,15 @@ def create(
             camera_obj.parent = asset_group
 
         return asset_group
+
+    def get_instance_attr_defs(self):
+        defs = lib.collect_animation_defs()
+
+        defs.append(
+            NumberDef("unitScale",
+                      label="Unit Scale (FBX)",
+                      default=1.0,
+                      tooltip="Scale of the model, valid only for FBX export.")
+        )
```

Comment:
If we need to repeat this for _each_ creator - shouldn't this be an attribute definition on the Extractor plugins instead?

Reply:
Ah I had this commented already here: https://github.com/ynput/ayon-core/pull/658#discussion_r1644003785